include/deal.II/grid/grid_in.h

```diff
@@ -767,6 +768,24 @@ class GridIn
   static std::string
   get_format_names();
 
+  /**
+   * Return a map containing field data associated with the elements of an
+   * external vtk format mesh imported using read_vtk().
+   * The format of the returned map is as
+   * follows:
+   * - std::string stores the name of the field data (identifier) as specified
+   * in the external mesh
+   * - std::vector<double> stores value for the given identifier in each cell.
+   * To access the value, use field_data[name_field][cell_id]. For example, if
+   * the vtk mesh contains field data 'Density' defined at the cells,
+   * field_data['Density'][0] would be the density defined at cell ID '0',
+   * which corresponds to index 0 of the vector. The length of the vector in
+   * field_data['Density'] would equal the number of elements in the coarse
+   * mesh.
+   */
+  const std::map<std::string, std::vector<double>> &
+  get_field_data() const;
```

Comment:
```suggestion
  get_cell_data() const;
```

Reply:
Elsewhere in the library we refer to these types of vectors as 'cell data': lets do the same thing here.